dockerfiles/README.md

```diff
@@ -1,37 +1,34 @@
 # Using Harmoni with Docker
 
-To launch the complete harmoni dev setup in docker:
-1. In order to run with window forwarding on linux use:
-```
-    xhost +local:
-```
+**Important** Please note that the kinetic build is not currently functional and the noetic build uses a kinetic w2l image because the w2l is not supported on 20.04. 
 
-2. Use docker compose to launch the complete system (will build if necessary, use --build to force)
-```
-    docker-compose -f docker-compose-harmoni-dev.yml up
+To launch the complete harmoni dev setup in docker:
+1. In order to run on linux use:
+```bash
+    sh run_compose.sh
 ```
 
-3. Launch the desired packages and run the desired scripts in the respective docker containers
+2. Launch the desired packages and run the desired scripts in the respective docker containers
 
-    For example, to use Wave2Letter (requires running get_w2l_models.sh first):
+    For example, to use Wave2Letter:
     - From harmoni_core, open a terminal and:
         - ```roscore```
 
     - From harmoni_hardware:
         - ``` roslaunch harmoni_microphone microphone_service.launch test:=true```
         - _Note: you can also use the alias ```rlhardwareservices``` we have provided, which will launch the hardware you have set in the configuration_
 
-    - From ros_w2l:
+    - From harmoni_w2l:
         - ```roslaunch harmoni_stt stt_example.launch```
          - _Note: you can also use the alias ```rlspeech``` we have provided_
 
 ## Why Docker?
 Although Harmoni will work without Docker, Harmoni is intended to be used with Docker to maximize portability and scalability. By using Docker Harmoni is quick and easy to set up on any OS or Hardware which currently supports docker.  We provide pre-built images for development, lightweight images for deployment, and even support for deploying to ARM chipsets.
 
-## Docker Containers
-We provide two sets of docker images. The first set is a heavier development set of images, which are based off of a ubuntu-16 image and contain graphical tools for use in a typical development cycle. These are intended to be helpful for developing interactively on a new machine. We also provide a set of lightweight images for deployment to space constrained machines, as well as images built with the arm architecture. 
+Docker also allows us to (mostly) mix and match OS/ROS versions in different containers. For instance we use noetic with vision containers because of the python3-opencv integration but we use kinetic with w2l because facebook doesn't support w2l on 20.04 yet. This works for communication with standard messages and harmoni messages but may cause issues with other message types. [See here for more information.]()
```

Comment:
```suggestion
Docker also allows us to (mostly) mix and match OS/ROS versions in different containers. For instance we use noetic with vision containers because of the python3-opencv integration but we use kinetic with w2l because facebook doesn't support w2l on 20.04 yet. This works for communication with standard messages and harmoni messages but may cause issues with other message types. [See here for more information.](https://answers.ros.org/question/45153/ros-fuerte-node-communicating-with-ros-electric-node/?answer=45160#post-id-45160)
```
One possible link. There are a bunch of similar questions on ROS answers if you look for them. They generally have the same answer (not supported, but often works)